avocado/utils/simulation.py

```diff
@@ -0,0 +1,395 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2013-2014
+# Author: Jan Scotka <jscotka@redhat.com>
+
+"""
+Module with simlation for various devices, network disc, etc
+"""
+
+import re
+import logging
+import time
+import process
+
+log = logging.getLogger('avocado.test')
+
+
+class NetworkBridge(object):
+    """
+    Create network bridge device
+    """
+
+    def __init__(self, brname=None):
+        """
+        Creates network bridge device with name 
+
+        :param brname: name of brdige device 
+        """
+        self.brname = brname
+        self.devicelist = set()
+        try:
+            process.run("brctl --help", verbose=False)
+        except:
+            raise BaseException(
+                "Unable to locate brdige-utils, please install them (yum install bridge-utils)")
+
+        if self.brname and self.__checkifbridgeexist():
+            log.debug("Bridge device ADD: " + self.brname)
+            self.__createbridge()
+
+    def getName(self):
```

Comment:
camelCase is not the preferred style for methods in Python, it'd be really nice to adjust it to match pep style...